Source/Scene/Primitive.js

```diff
@@ -1203,7 +1204,14 @@ var numberOfCreationWorkers = Math.max(
   1
 );
 var createGeometryTaskProcessors;
-var combineGeometryTaskProcessor = new TaskProcessor("combineGeometry");
+function removeGeometryTaskProcessors() {
+  if (createGeometryTaskProcessors) {
+    for (var i = 0; i < createGeometryTaskProcessors.length; i++) {
+      createGeometryTaskProcessors[i].destroy();
```

Comment:
Are you sure that this is a safe way to destroy the `createGeometryTaskProcessors` object?

Reply:
I call this function after workers complete so if worker complete and then values taken we dont need worker anymore.